guides/doc-Deploying_Project_on_AWS/topics/AWS_Installing_Capsule_on_AWS.adoc

```diff
@@ -5,4 +5,4 @@ On your AWS environment, complete the following steps:
 
 . Connect to the new instance.
 . Install {SmartProxyServer}.
-For more information, see {InstallingSmartProxyDocURL}[{smart-proxy-installation-guide-title}].
+For more information, see {InstallingSmartProxyDocURL}[{InstallingSmartProxyDocTitle}].
```

Comment:
IMHO missing `_` to make doc title italic. cc @Lennonka

Reply:
There are many more `_` missing in that doc. So I'd prefer to skip fixing it in this PR?

Reply:
You certainly can! I've put it on my "look at this shiny stuff" pile (and I won't tell you how big it is :sweat_smile: )